staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
Since each stage sends a separate event object and CreationTimestamp is in the metadata of the object, I would argue that this field makes sense when set with `time.Now()` on each stage

I think that either way what `Timestamp` means is not very clear, it should be something like `RequestStartTimestamp` or `StartTimestamp`